### PR TITLE
fix(wallet-setup): wallet list spacing

### PR DESCRIPTION
**Pull request by banklesss**
[YOMO-1308](https://emurgo.atlassian.net/browse/YOMO-1308)

[YOMO-1308]: https://emurgo.atlassian.net/browse/YOMO-1308?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx [hunk 1]
```diff
@@ -119,6 +120,8 @@ export const SelectWalletFromList = () => {
         <Space height="m" />
 
         <OnlyDevButton />
+
+        <Space height="l" />
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx:124]: REAL FIX HERE @stackchain @michaeljscript  @jorbuedo

Comment (banklesss) [reply apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx:124]: the rest is just moving folders and files

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx (hunk 1 above)
Comment (stackchain) [reply apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx:124]: I think it was already moved in dev, maybe rebase and apply only this update

Comment (banklesss) [reply apps/wallet-mobile/src/features/WalletManager/SelectWalletFromList/SelectWalletFromListScreen.tsx:124]: its a new refactor. Wallet list -> to WalletManager folder

not changed in develop yet https://github.com/Emurgo/yoroi/tree/develop/apps/wallet-mobile/src/features/SetupWallet/useCases/SelectWalletFromList

**Pull request merged**